2차/윤형진_시저암호.py

```diff
@@ -0,0 +1,14 @@
+def solution(s, n):
+    answer = ''
+    for i in s :
+        if i != " " :
+            if (ord(i) + n > 90 and ord(i) + n < 97) or (ord(i) + n > 122) : #대문자에서 Z를 초과한 경우
+                answer = answer + (chr(ord(i) + n - 26))
+            else :
+                answer = answer + (chr(ord(i) + n))
+        else :
+            answer = answer + " "
```

Comment:
이 부분도 구조 분해를 통해 로직을 단순화 하면 좋을 것 같습니다.

```suggestion
            if (ord(i) + n > 90 and ord(i) + n < 97) or (ord(i) + n > 122) : #대문자에서 Z를 초과한 경우
                answer = answer + (chr(ord(i) + n - 26))
                continue

            answer = answer + (chr(ord(i) + n))
            continue
        answer = answer + " "
```